jobs/designer.md

```diff
@@ -0,0 +1,64 @@
+
+**Salary:** 2k-3k USD gross per month.
+
+
+## What will you do:
+- Design user-friendly interface for clients’ web and mobile applications.
+- Build UX part of the apps on the basis of the existing design system.
+- Create the whole design system for the internal products by a demand using AI tools.
+- Help other teams with design-related tasks: design website pages, edit photos’ size and shape, etc.
+- Manage and coach the design team.
+
+
+## What qualities do we expect you to have:
+- High level of expertise in UX.
+- Independency and ability to work without teammate’s guidance. We’re starting to build our design team, so some time you’ll work as a single designer in the department.
+- Fluency in English (writing and speaking) for daily use.
+- Understanding of the principles of building and managing the team.
+- Proactivity: ability to see points for improvement and initiativeness in proposing changes.
+- Versatileness: you’ll work on the client’s project, but sometimes we can ask you to help the team with internal tasks.
+- Interest in working with AI tools.
+
+## Would be a plus:
+- Knowledge of UI Kit, React, HTML.
+
+
+## What do we offer:
+- Transparent work culture.
+- 50% compensation for self-development activities: courses, conferences, books, educational platforms, English, and sports.
+- Remote work and flexible schedule.
+- 32 paid days off per year – you can use them for vacations, occasional days off, and public holidays, only workdays count.
+- Full coverage of sick leaves.
+- 200 USD per year for medical expenses.
+- 500 USD per year for home office equipment.
+  
+
+## About us
+
+[![datarockets team](https://github.com/datarockets/career/blob/master/images/photo%20from%20New%20Year%20party.3.jpg)](https://www.instagram.com/datarockets/)
+
+
+datarockets is a [remote](https://github.com/datarockets/career#remote) software development company focusing on setting up transparent and honest processes with the team and clients. We adore flexibility, freedom and independence, so we have no micromanagement and draining bureaucracy. 
+At datarockets, the medium project duration is 12 months. With most of the clients, we work for years. Being a relatively small company, we have the freedom to pick the projects that we feel most passionate about, which benefits both our clients and us. We have experience building apps serving a variety of industries, such as FinTech, IoT, Health & Fitness, e-commerce, and many more. Among our clients: YC-backed startups, Individual Entrepreneurs, Mid-sized companies, Public Institutions, and Series E startups.
+
+Check out our resources for more information:
+
+- [Case studies](https://datarockets.com/case-studies/) to learn more about the types of projects we work with.
+- [Company Codex](https://github.com/datarockets/career#work-as-a-team-with-clients) to see if we share the same values.
```

Comment:
This link doesn't lead to our company's codex but to some other page. I think we should use this one :)
```suggestion
- [Company Codex](https://datarockets.com/blog/company/datarockets-codex-core-values/) to see if we share the same values.
```